### ISSUE TITLE
anchor tags missing title

### ISSUE THREAD

**Issue by marlowecheng** [marlowecheng/marlowes-portfolio#10]
**Status:** Open

**Description:** Footer links are missing the title attribute when hovering over them. This is important for accessibility.

**Steps to reproduce:** 
1. Hover over email and Discord icons

**Expected result:** When hovering over clickable icons, the title attribute content should show up.

**Actual result:** When hovering over clickable icons, nothing shows up due to lack of title attribute.

**Configuration:** All platforms

**Severity:** Critical

**Priority:** Critical

**Type:** SEO, Accessibility

**Comment by marlowecheng** [comment]
Added title attributes to footer.php links.
<img width="1337" alt="image" src="https://user-images.githubusercontent.com/114950923/227379596-8312d343-36af-4d41-8264-cac074afc116.png">
